<?xml version="1.0"?>
<launch>

  <node name="octomap_server" pkg="octomap_server" type="octomap_server_node" output="screen">
    <param name="frame_id" value="base_footprint"/>
    <param name="resolution" value="0.01"/>
    <param name="latch" value="false"/>
    <param name="sensor_model/max_range" value="3.0"/>
    <param name="pointcloud_min_z" value="0.1"/>
    <param name="pointcloud_max_z" value="1.0"/>
    <param name="occupancy_min_z" value="0.1"/>
    <param name="occupancy_max_z" value="1.0"/>
    <param name="sensor_model/hit" value="0.9"/>
    <param name="sensor_model/miss" value="0.1"/>
    <remap from="cloud_in" to="/camera/depth/color/points"/>
  </node>

</launch>
